yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -544,6 +545,7 @@ private[spark] class Client(
       logInfo(s"Credentials file set to: $credentialsFile")
       val renewalInterval = getTokenRenewalInterval(stagingDirPath)
       sparkConf.set("spark.yarn.token.renewal.interval", renewalInterval.toString)
+      SparkHadoopUtil.get.startExecutorDelegationTokenRenewer(sparkConf)
```

Comment:
I would rather move this somewhere else.  It doesn't really have anything to do with setting up the launch env.   Perhaps move it to setupCredentials()